.ddev/config.yaml

```diff
@@ -46,7 +46,7 @@ hooks:
 # This allows ddev to manage its own settings.ddev.php file.
 disable_settings_management: false
 
-type: drupal10
+type: drupal11
```

Comment:
This change will require DDEV >=1.24.0

Reply:
I updated our pre-start hook that checks the ddev version number.